### PR TITLE
Debug Information's sending to Trinket

### PR DESCRIPTION
**Pull request by RedstoneAlmeida**

### CODE REVIEW

File: src/Trinket/EventListener.php [hunk 1]
```diff
@@ -33,25 +33,29 @@ public function onChat(PlayerChatEvent $ev) {
...
-        $pk->id = Info::TYPE_PACKET_SERVER_INFORMATION;
-        $pk->data = json_encode([
-            "type" => 0,
-            "maxplayers" => $event->getMaxPlayerCount(),
-            "online" => $event->getPlayerCount(),
-            "motd" => Server::getInstance()->getNetwork()->getName(),
-            "serverId" => $this->data["name"],
-        ]);
-        $this->plugin->getPacketQueue()->push($pk);
+        if($this->data["queryUpdate"] === true or "true") {
```
Comment (ImagicalGamer) [inline src/Trinket/EventListener.php]: if(boolval($this->data["queryUpdate"]) === true)

File: src/Trinket/EventListener.php [hunk 2]
```diff
@@ -33,25 +33,29 @@ public function onChat(PlayerChatEvent $ev) {
...
 
     public function onLogin(PlayerLoginEvent $event) {
-        $pk = new DataPacket();
-        $pk->id = Info::TYPE_PACKET_SERVER_INFORMATION;
-        $pk->data = json_encode([
-            "type" => 1,
-            "message" => "{$event->getPlayer()->getName()} connected to server {$this->data["name"]}",
-        ]);
-        $this->plugin->getPacketQueue()->push($pk);
+        if($this->data["playerJoined"] === true or "true") {
```
Comment (ImagicalGamer) [inline src/Trinket/EventListener.php]: if(boolval($this->data["playerJoined"]) === true)

**Comment by RedstoneAlmeida** [comment]
Ready

**Comment by ImagicalGamer** [comment]
Also change TYPE_PACKET_SERVER_INFORMATION to TYPE_PACKET_CHAT on PlayerJoinEvent and make sure the data value is the message not an array.

**Comment by ImagicalGamer** [comment]
ServerInformationPackets are only for QueryRegenerateEvent?

**Comment by RedstoneAlmeida** [comment]
ServerInformationPacket, use for transfer any informations

**Comment by ImagicalGamer** [comment]
Alright.

**Pull request merged**